KomodoPlatform/atomicDEX-Desktop
# Min vol bug

closes https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/2183

To test:
- go to pro view with a pair which you have low balance (< minvol / 4)
- If balance at 25, 50 or 100 % would be less than min vol, these buttons should show as "min" (and if 50% of balance is < minvol, there should only be "min" or "max" button"
- Confirm clicking buttons returns sane values.
